### ISSUE TITLE
Allow hibf in ganon?

### ISSUE THREAD

**Issue by SannaAb** [nf-core/taxprofiler#401]
### Description of feature

Thank you for a really nice tool. 

I was going to try out ganon and was generating a database using their recommendation for refseq https://pirovc.github.io/ganon/default_databases/. 

```
ganon build --source refseq --organism-group archaea bacteria fungi viral --threads 48 --hibf --level species --db-prefix abfv_rs
```

When using it in taxprofiler i got errrors, I realized it was due to taxprofiler looking for ibf while i have hibf. I regenerated the database without the hibf but when testing it kept crashing due to depleted memory (running alone in a node with 512 GiB ram using an insilco sample of 6 species with 1M reads in total). Using the hibf version it runs fine. 

Would it be possible to include hibf for ganon in taxprofiler?

**Comment by jfy133** [comment]
Nope, no reason not to include that! We just need to update a search pattern in the Module :), should be easy

**Comment by SannaAb** [comment]
Awsome, when you looking into it can you also update ganon to the newest version? I was trying to use the tax flag but it seems it is not available for 1.5.1.

**Comment by jfy133** [comment]
Yup! That's planned 👍

**Comment by pirovc** [comment]
Just a heads-up, a new ganon version (2.0) is planned to be out soon(ish) and there will be some parameter changes, where `--hibf` will vanish since it will be the default behavior (there will be a `--ibf` option then). I don't think this should stop this request but a newer ganon version may be out before you fix this.

**Comment by sofstam** [comment]
@pirovc Thanks for letting us know!

**Comment by pirovc** [comment]
ganon2 (ganon v2.0.0) is now released. The new behavior is to generate the `.hibf` by default with `ganon build` and `ganon build-custom`. The option to switch to the old filter type `.ibf` is now `--filter-type ibf`. Anyways, when using the database with `ganon classify` the extension does not matter, the tool will auto-detect the filter type. I hope that clarifies and simplifies the issue. There was also some other changes in parameters and output files https://github.com/pirovc/ganon/releases/tag/2.0.0

Would be great to have ganon2 in the taxprofiler, let me know if any help is needed.

**Comment by jfy133** [comment]
Whoop! That's awesome, thanks @pirovc ! 

I will aim to update the nf-core module soonish (disclaimer: I'm on parental leave and as of this morning baby is sick, and Kita is shut for the rest of week 🙄 perfect combination). I'll ping you for review of the modules to make sure I've not missed any mandatory options 👍

Sorry for the slow progress on this @SannaAb will get there eventually

**Comment by jfy133** [comment]
Oh and @pirovc always feel free to let us know if you feel like learning some nextflow or nf-core ;) we are happy to help you get involved 😉

**Comment by sofstam** [comment]
The ganon is updated to version 2. Maybe we can close this issue? @jfy133 @LilyAnderssonLee
